imap_processing/ena_maps/ena_maps.py

```diff
@@ -634,15 +634,10 @@ class LoPointingSet(PointingSet):
 
     def __init__(self, dataset: xr.Dataset):
         super().__init__(dataset, spice_reference_frame=geometry.SpiceFrame.IMAP_DPS)
-        # TODO: Use spatial_utils.az_el_grid instead of
-        #  manually creating the lon/lat values
-        inferred_spacing_deg = 360 / dataset.longitude.size
-        longitude_bin_centers = np.arange(
-            0 + inferred_spacing_deg / 2, 360, inferred_spacing_deg
-        )
-        latitude_bin_centers = np.arange(
-            -2 + inferred_spacing_deg / 2, 2, inferred_spacing_deg
-        )
+        # 0.1 degree bin spacing in both spin angle (longitude) and off angle (latitude)
+        bin_spacing_deg = 0.1
+        longitude_bin_centers = np.arange(0 + bin_spacing_deg / 2, 360, bin_spacing_deg)
+        latitude_bin_centers = np.arange(-2 + bin_spacing_deg / 2, 2, bin_spacing_deg)
```

Comment:
Can't these be grabbed from the PSET Dataset?
```suggestion
        longitude_bin_centers = self.data["spin_angle"].data
        latitude_bin_centers = self.data["off_angle"].data
```

Reply:
I reworked this to use the hae_longitude/latitude variables now instead, grabbing those directly from the PSET. (brought in a previous commit I had into this PR)